mubench.reviewsite/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This script should be called `run_all_tests` or something. `run` does not really reflect its behaviour.